src/DynamoCoreWpf/Views/Menu/PreferencesView.xaml

```diff
@@ -978,17 +978,74 @@
             <Grid.RowDefinitions>
                 <RowDefinition Height="*"></RowDefinition>
             </Grid.RowDefinitions>
-            <Label Name="SavedChanges" 
+            <Grid>
+                <Grid Grid.Column="0" HorizontalAlignment="Left" Margin="14,0,0,0">
+                    <Grid>
+                        <Grid.RowDefinitions>
+                            <RowDefinition></RowDefinition>
+                            <RowDefinition></RowDefinition>
+                        </Grid.RowDefinitions>
+                        <Grid Grid.Row="0">
+                            <Grid>
+                                <Grid.ColumnDefinitions>
+                                    <ColumnDefinition Width="16"/>
+                                    <ColumnDefinition Width="*" />
+                                </Grid.ColumnDefinitions>
+                                <TextBlock Name="importLabel"
+                                    Grid.Column="1"
+                                    Text="{x:Static p:Resources.ImportPreferencesText}"
+                                    Style="{StaticResource TextBlockLink}"
+                                    HorizontalAlignment="Left"
+                                    VerticalAlignment="Center"
+                                    Margin="5,-7,0,0"
+                                    MouseLeftButtonDown="importTextBlock_MouseLeftButtonDown"/>
+                                <Image Name="importToolTipIcon"
+                                    Grid.Column="0"
+                                    Margin="0,-5,0,0"
+                                    Width="16"
+                                    Height="16"
+                                    HorizontalAlignment="Center"
+                                    VerticalAlignment="Center"
+                                    Style="{StaticResource QuestionIcon}"
+                                    ToolTipService.ShowDuration="10000">
```

Comment:
If this is intentional, I wonder if this should be moved into `Style="{StaticResource GenericToolTipLight}"`

Reply:
I have tried adding the ShowDuration in the GenericToolTipLight style but it didn't work, we need to define it per each kind of control that use a Tooltip, after researching the only way to apply a generic ShowDuration in the app is in the program main as the below image, Please let me know if I apply this change or just keep the current one setting an explicit duration to that image like other places doing. 

![imagen](https://user-images.githubusercontent.com/96534278/189392626-b6a22639-8c8b-41bb-b6fe-c2cd65b72d1e.png)

![imagen](https://user-images.githubusercontent.com/96534278/189392666-916bdeaa-4e37-4e67-ad88-24ec5b773f17.png)

Reply:
Can you remind me the default value of this setting? If not too different, I think the current implementation would be fine based on your investigation

Reply:
5 seconds as I recall, is fall short time to read the tooltip message, that's why I'm setting it at 10 seconds.